### ISSUE TITLE
Error on startup

### ISSUE THREAD

**Issue by EldRoos** [sebgroup/react-components#77]
**Describe the bug**

Cannot use version 2.0.0 or newer in our project.

When starting project get this error:
[...]/node_modules\@sebgroup\react-components\dist\Slider\Slider.js:1:3809: Unexpected token (1:3809)

**To Reproduce**
1. Got to https://github.sebank.se/SEBHome/Loans.Mortgages.Portfolio.UI
2. Clone project to your local (and go to that location in console).
3. Change package.json "@sebgroup/react-components": "^1.1.0", to "^2.0.3".
4. Type "npm install" & "npm start" in console.
5. See error of above.

**Expected behavior**
Not getting any error.

**Desktop (please complete the following information):**
 - OS: Win 10.
- Console: Cmder.

**Additional context**

If I check Slider.js, it's the '#' that is the character at number 3809.
(o+="\n/*# sourceMappingURL=data:

We currently only use the slider from react-components.

**Comment by DanSallau** [comment]
A quick peek into your package.josn shows that you are running an outdated version of this component. 

![image](https://user-images.githubusercontent.com/12234030/67271858-fcf74200-f4ed-11e9-8d4d-62361c536995.png)


Can you please update it like below.
```

npm uninstall @sebgroup/react-components
npm install @sebgroup/react-components@latest --save
```

Of course your react version too might require upgrade because the latest version requires support for hooks which i think your react version doesn't support . Please install `react 16.8.6 `and above . So basically, an upgrade to your dependecies and this module should fix your problem. Please let us know if you are having problem with the update :)

**Comment by EldRoos** [comment]
Ah of course, sorry I forgot to add that step. And yes project works on old version but not new.

3. Change package.json "@sebgroup/react-components": "^1.1.0", to "^2.0.3".

**Comment by DanSallau** [comment]
@EldRoos my mistake :) . We cloned your project already and are looking into it.

**Comment by omarVodiaK** [comment]
@EldRoos we found the issue with the sourcemap, it is not related to react-components but related to parcel dependency
Adding this to your package.json should fix the error issue that you are facing 

`"start": "xcopy /y config-local.json dist\\ && parcel --no-source-maps index.html --port 6102",`

You can read more about it in this [url](https://github.com/parcel-bundler/parcel/issues/2408)
According to Parcel a fix will be released in version 2.0, unfortunately you can't cherry pick the fix without creating complexity and inconsistency.

Please let us know if you still facing other issues after adding --no-source-maps

**Comment by EldRoos** [comment]
Yes that fixed it. Thank you so much.
